Say/Say.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Say.swift
+//  Say
+//
+//  Created by Seungwon Kim on 2016. 9. 24..
+//  Copyright © 2016년 youknowone.org. All rights reserved.
+//
+
+import Cocoa
+
+/**
+    Easy-to-use interface for `say` command in OS X.
+*/
+class SayAPI: NSObject {
+    let speechSynthesizer: NSSpeechSynthesizer?
+    var text: String
+    var voice: VoiceAPI?
+    
+    var outputFile: String? = nil
+    
+    override var description: String {
+        get {
+            return "<Say: '\(self.text)'>"
+        }
+    }
+
+    /**
+        Construct a say interface with given text and voice.
+     
+        - Parameters:
+        - text: A text string to composite speech.
+        - voice: A voice to composite speech. If given voice is nil, default voice is used.
+     */
+    init(text: String, voice: VoiceAPI?) {
+        speechSynthesizer = NSSpeechSynthesizer.init()
```

Comment:
This line doesn't use `self.` indicators not like other 2 lines.
For integrity and readability, I recommend to use `self.` indicator everywhere.